providers/parameter.rb

```diff
@@ -26,7 +26,7 @@
 use_inline_resources if defined?(:use_inline_resources) # ~FC113
 
 def parameter_exists?(vhost, name)
-  cmd = if rabbitmq_37? && !use_distro_version? && node['rabbitmq']['version'] >= '3.7.10'
+  cmd = if Gem::Version.new(installed_rabbitmq_version) >= Gem::Version.new('3.7.10')
```

Comment:
Ugh, this… deserves a comment at least. Is it common to use `Gem::Version` that way in Chef cookbook? :)

Reply:
You are totally right, i put a comment (but not an explanation) about it in the commit message, since i thought replicating the same comment in all places where this is used it not ideal. I googled a lot what is a good way for version comparison in ruby, after i realised that "3.7.8" >= "3.7.10" returns true during the refactoring. The only two things i found are Gem::Version.new() and Semantic::Version.new(), while the first one is already build in and the latter need a gem. So i chose to go for the first one. Do you think i should add a comment/explanation about it into the providers code?

Reply:
Nah, it's probably more or less obvious and indeed relies on what's in the standard library in modern Ruby versions. I'm happy to leave this as it is.